.github/workflows/documentation.yml

```diff
@@ -2,8 +2,6 @@ name: Publish Documentation
 
 on:
   push:
-    branches:
-    - 'main'
```

Comment:
I think we still want to build the docs on main (currently available at this URL: https://rasa.com/docs/rasa/next/)